Library_group9/libraryAutomation_group9.5/libraryAutomation_group9.5.iml

```diff
@@ -1,66 +1,66 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
please add .iml file to gitignore, along with .idea and target folder, then,,,
Clean your files up by running Maven clean.
Then, run cukesrunner again to generate a new clean target folder.
Update all branches and merge them into one another.
Push an amended commit to your remote.